libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -128,7 +128,7 @@ library
     bytestring,
     cardano-crypto,
     cardano-crypto-class,
-    cardano-crypto-wrapper,
+    cardano-crypto-wrapper >=1.6,
```

Comment:
There should be no bounds in `cardano-ledger-test` for packages that live in the same repo. It will always use the package that is in the repo regardless what bounds are specified, unless they do create a conflict.
So, please remove it

Reply:
Oops! That was a mistake. I'm not sure how that snuck in.

I think it's not just that it lives in the same repo but also that it also doesn't get released to CHaP to there's no possibility of building it without the matching `cardano-crypto-wrapper`.